include/common.h

```diff
@@ -43,6 +43,9 @@ typedef uint16_t ioaddr_t;
 struct DynamicConfig {
   bool ignore_illegal_mem_access;
   bool debug_difftest;
+#ifdef CONFIG_STORE_LOG
```

Comment:
Probably we should not put `bool enable_store_log;` under a macro for this struct here.

When multiple macros are included, we cannot determine their positions/offsets in the struct from external code. Different configs may affect this struct, and then difftest may not see the same struct type (to update the config here).

It looks better if `enable_store_log` is always defined but not used when the macro is not defined

Reply:
Sure. I will fix it